doc-content/jbpm-docs/src/main/asciidoc/BPMN2/design-business-process-proc.adoc

```diff
@@ -29,5 +29,91 @@ endif::[]
 . In the upper-right corner, click the *Diagram properties* image:processes/diagram_properties.png[] icon and add your business process property information, such as process data and variables:
 .. Scroll down and expand *Process Data*.
 .. Click image:processes/btn_plus.png[] next to *Process Variables* and define the process variables that you want to use in your business process.
+
++
+--
+.General process properties
+[cols="30%,70%", options="header"]
+|===
+|Label
+|Description
+
+| *Name*
+| Enter the name of the process.
+
+| *Documentation*
+| Describes the process. The text in this field is included in the process documentation, if applicable.
+
+| *ID*
+| Enter an identifier for this process, such as `orderItems`.
+
+| *Package*
+| Enter the package location for this process in your {PRODUCT} project, such as `org.acme`.
+
+| *ProcessType*
+| Specify whether the process is public or private (or null, if not applicable).
+
+| *Version*
+| Enter the artifact version for the process.
+
+| *Ad hoc*
+| Select this option if this process is an ad hoc subprocess. (Currently not supported.)
```

Comment:
Can you please double-check this? I think we support this already.

Reply:
Done, removed the bracket.